tensorboard/main.py

```diff
@@ -135,6 +135,15 @@ def create_tb_app(plugins):
       plugins=plugins)
 
 
+def _quiet_werkzeug_logging():
+  logger = base_logging.getLogger('werkzeug')
+  logger.setLevel(base_logging.WARNING)
+  #
+  # Werkzeug will only add logging handlers to its logger if we haven't
+  # touched it. Thus, we have to add back the standard handlers.
+  logger.addHandler(base_logging.StreamHandler())
```

Comment:
I like having this be explicit and I'm not sure it merits a comment. In fact, I'd even recommend inlining this function.